src/app/pages/all-volunteers/all-volunteers.component.html

```diff
@@ -16,7 +16,7 @@
                 <tr>
                   <th scope="col">FFFFirst Name</th>
                   <th scope="col">Last Name</th>
-                  <th scope="col">email</th>
+                  <th scope="col">email</th>  
```

Comment:
Change the label of it as Action.

Reply:
Also make sure you are adding more column. IT looks like there are only three columns still. We want four